tests/Mpdf/Color/ColorConverterTest.php

```diff
@@ -89,6 +89,8 @@ public function convertColorsProvider()
 			['inherit', false],
 			['transparent', false],
 
+            ['#CCC url("img_tree.gif")', "3\xcc\xcc\xcc\x00\x00"],
```

Comment:
Didn't see any test case which tests something like this. I saw it in one of the comments in the source and included it.